src/Orchard.Cms.Web/Modules/Orchard.OpenId/Views/Access/Authorize.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@using Orchard.OpenId.ViewModels
+@model AuthorizeViewModel
+
+<div class="jumbotron">
+    <h1>Authorization</h1>
+    
+    <p class="lead text-left">@T["Do you want to grant <strong>{0}</strong> access to your data? (scopes requested: {1})", @Model.ApplicationName, @Model.Scope]</p>
+
+    <form method="post">
+        @Html.AntiForgeryToken()
+
+        <input type="hidden" name="request_id" value="@Model.RequestId" />
```

Comment:
No, it doesn't work. And the explanation I imagine it is related with this https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/blob/54701d96841007ec11dbb5d24abae88b5df65e14/src/AspNet.Security.OpenIdConnect.Primitives/OpenIdConnectConstants.cs#L221